lib/Onyx.js

```diff
@@ -788,7 +887,7 @@ function clear() {
  */
 function mergeCollection(collectionKey, collection) {
     // Confirm all the collection keys belong to the same parent
-    _.each(collection, (data, dataKey) => {
+    _.each(collection, (_data, dataKey) => {
```

Comment:
@marcaaron This looks suspicious to me, but it was end of a day for me so I opted was quick solution. Sorry about that.

Reply:
I see now, had this open and was investigating why is this happening and havent seen the new comments